heroic-elasticsearch-utils/src/main/java/com/spotify/heroic/elasticsearch/Connection.java

```diff
@@ -141,6 +142,7 @@ public SearchScrollRequestBuilder prepareSearchScroll(String scrollId) {
         return client.getClient().prepareSearchScroll(scrollId);
     }
 
+    @Nonnull
```

Comment:
Doesn't this annotate the return type rather than the argument?

I'm also not as concerned with the `Nonnull` annotation - I think our general approach has been to assume non-null, and do checking in places where its explicitly allowed (except for Kotlin classes, since its always explicit there).

Reply:
then I can get rid of the annotation since I check for it before calling this method, right?

Reply:
yeah sgtm

Reply:
👍 removed